lite-rpc/src/cli.rs

```diff
@@ -46,6 +46,10 @@ pub struct Config {
     pub grpc_addr: String,
     #[serde(default)]
     pub grpc_x_token: Option<String>,
+    #[serde(default)]
```

Comment:
Make it Vec<String> instead of adding another argument.
Example : 
```
#[arg(short, long, value_delimiter = ',')]
 pub grpc_addr: Vec<String>,

```

Reply:
reworked it, but not as vector .. that's really an headache